src/web/resources/report.js

```diff
@@ -275,16 +272,18 @@ const CostAccuracy = new Component('#cost-accuracy', {
         for (let test of results_json.tests) {
             if (test.name == this.elt.dataset.benchmarkName) {
                 let [initial_pt, best_pt, rest_pts] = test["cost-accuracy"];
-                let target_pt = test["target"] && [this.elt.dataset.targetCost, test["target"]]
+
+                let target_pts = test["target"]
                 rest_pts = [best_pt].concat(rest_pts)
-                $svg.replaceWith(await this.plot(test, initial_pt, target_pt, rest_pts));
-                $tbody.replaceWith(await this.tbody(test, initial_pt, target_pt, rest_pts));
+
+                $svg.replaceWith(await this.plot(test, initial_pt, target_pts, rest_pts));
+                $tbody.replaceWith(await this.tbody(test, initial_pt, target_pts, rest_pts));
```

Comment:
Not sure I see a difference? Just some indentation stuff?

Reply:
A combination of extra lines, `target_pt` -> `target_pts`, and other neighboring changes.